julearn/scoring/available_scorers.py

```diff
@@ -57,35 +57,31 @@ def register_scorer(name, scorer, overwriting=None):
     scorer : callable
         function of signature (estimator, X, y) see:
         https://scikit-learn.org/stable/modules/model_evaluation.html#scoring
-    overwriting : bool | None, optional
-        decides whether overwriting should be allowed, by default None.
+    overwrite : bool | None, optional
+        decides whether overwrite should be allowed, by default None.
         Options are:
 
-        * None : overwriting is possible, but warns the user
-        * True : overwriting is possible without any warning
-        * False : overwriting is not possible, error is raised instead
+        * None : overwrite is possible, but warns the user
+        * True : overwrite is possible without any warning
+        * False : overwrite is not possible, error is raised instead
     """
     if name in list_scorers():
```

Comment:
Same logic issue here.

The last else should not be there and `_extra_available_scorers[name] = scorer` should happen anyways.

If you see this code, if overwrite is `None`, nothing happens.

Reply:
Thx. was a stupid mistake I copied from the other one.